packages/data-types/src/types/versions/v1/geopoint.ts

```diff
@@ -0,0 +1,24 @@
+import { FieldType } from 'xlucene-evaluator';
+import BaseType from '../base-type';
+import { ElasticSearchTypes } from '../../../interfaces';
+
+export default class GeoPointType extends BaseType {
+    toESMapping(_version?: number) {
+        return { mapping: { [this.field]: { type: 'geo_point' as ElasticSearchTypes } } };
+    }
+
+    // TODO: need notion of injecting custom types, what about duplicates
+    toGraphQL() {
+        const customType = `
+            type GeoPointType {
```

Comment:
I think now is a good time to namespace the types (see comment above)

```suggestion
            type DTGeoPointV1 {
```